ansible-allinone-demo-vm/roles/opencast-allinone/templates/opencast-ingest.sh

```diff
@@ -12,6 +12,17 @@ do
   curl -O "https://data.lkiesow.io/opencast/test-media/${media}"
 done
 
+# Download additonal open licensed media (Public Domain or Creative Commons)
+# PD
+curl -O https://upload.wikimedia.org/wikipedia/commons/0/00/NASA%27s_new_High_Dynamic_Range_Camera_Records_Rocket_Test.webm
```

Comment:
Getting things from several sources makes a failure more likely.
What about allowing a command to fail in here?
That should probably be okay.